dvc/command/gc.py

```diff
@@ -94,6 +83,12 @@ def add_parser(subparsers, parent_parser):
     gc_parser.add_argument(
         "-r", "--remote", help="Remote storage to collect garbage in."
     )
+    gc_parser.add_argument(
+        "--remove",
+        nargs="*",
+        choices=supported_removal_filters,
+        help="Available choices for remove: %(choices)s",
```

Comment:
I guess this `help` should describe different choices, not just list them.

Reply:
Okay, I'll update this when the PR is ready. Just making sure I haven't broken anything.